.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: "${{ matrix.os }}"
     strategy:
       matrix:
-        os: [ubuntu-24.04, ubuntu-22.04]
+        os: [ubuntu-24.04, ubuntu-22.04, ubuntu-24.04-arm, ubuntu-22.04-arm]
```

Comment:
IMHO this is too much. to save some CPU and power please remove the 22.04-arm one. thanks!

Reply:
Will do